include/unifex/stop_immediately.hpp

```diff
@@ -413,6 +415,10 @@ struct _stream<SourceStream, Values...>::type {
     operation<Receiver> connect(Receiver&& receiver) && {
       return operation<Receiver>{stream_, (Receiver &&) receiver};
     }
+    template<typename Receiver>
+    operation<Receiver> connect(Receiver&& receiver) const& {
+      return operation<Receiver>{stream_, (Receiver &&) receiver};
+    }
```

Comment:
The cleanup_sender should not be const-lvalue connectable.
The cleanup operation has side-effects that are not idempotent and starting it multiple times would have undefined-behaviour.